README.md

```diff
@@ -0,0 +1,8 @@
+# svg_play
+## Making pretty generated SVGs in Python
+
+I am giving a talk at PyconAU 2019 showng how to build SVG and using generative art as a motivator. 
+
+This is where I'm puytting all my code so people can have a look, have a play and maybe even give advice (particularly on dumb if statements when I should be using clever maths).
+
+Everything is available to steal and play with . **Please share alike**.
```

Comment:
Please include a real LICENSE file like Apache2. It's really hard to use code like this as a professional without being confident of my rights.